data/json/items/armor/hats.json

```diff
@@ -269,6 +269,32 @@
       }
     ]
   },
+  {
+    "id": "hat_field",
+    "type": "ARMOR",
+    "name": { "str": "patrol cap" },
+    "description": "A simple round cap in woodland camouflage. Often used by the military when helmets arent needed. Its brim helps keep the sun out of your eyes.",
```

Comment:
```suggestion
    "description": "A simple round cap in woodland camouflage.  Often used by the military when helmets aren't needed.  Its brim helps keep the sun out of your eyes.",
```

Reply:
Thank you for calling that out, didn't have variants in mind when writing that description.